aws/main.tf

```diff
@@ -5,9 +5,9 @@ module "vpc" {
   name = "granica-vpc"
   cidr = "${var.vpc_cidr_prefix}.0.0/16"
 
-  azs             = ["${var.aws_region}a", "${var.aws_region}b", "${var.aws_region}c"]
-  private_subnets = ["${var.vpc_cidr_prefix}.0.0/20", "${var.vpc_cidr_prefix}.16.0/20", "${var.vpc_cidr_prefix}.32.0/20"]
```

Comment:
Why cut subnet count for private_subnets. It increases our chances to find Spot

Reply:
otherwise +1

Reply:
There is a bug in KRY.
When subnets are specified, we also need to specify az_ids (in the same order) - else terraform apply fails. Right now, it determines 2 az_ids automatically and uses them. This does not match the 3 subnets.

This can be fixed so as to get the az_ids from specified subnets.